### DISCUSSION TITLE
Is there a JSON Schema for Lexical state?

### DISCUSSION THREAD

**Discussion by fdemian** [facebook/lexical#4955] [Q&A]
For context, I'm currently exporting lexical editor state as JSON and saving it on my application. I'm looking for a way to detect and reject any content that is not a valid lexical editor state on the server. 

Is there a JSON sechema that describes a valid lexical content state?

**Comment by moy2010** [comment]
> Is there a JSON sechema that describes a valid lexical content state?

No. The JSON schema of your editor state is specific to your domain, so it's only you who's responsible of describing/validating it.
